src/rich-text-editor.component.js

```diff
@@ -190,9 +191,17 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   }
 
   function setHTML(html) {
-    emptyEditor()
-    divRef.current.innerHTML = richTextContext.sanitizeHTML(html, 'setHTML')
-    focus()
+    if(!initialMount.current) {
```

Comment:
We don't need the initialMount ref. We already have a state value called `hasSetInitialHTML`.

Reply:
Pre-empting another review comment -- I think it would be better to pass in whether it's the first mount as an optional boolean value to the setHTML function. See

https://github.com/CanopyTax/bandicoot/blob/b541feda5917932b4f66b5f19c8173446c91d1d9/src/rich-text-editor.component.js#L123-L124

^ we cannot depend on hasSetInitialHTML to be set in time before the setHTML runs. So we should just pass in `true` as an additional argument to setHTML imo

Reply:
is there any reason we shouldn't just invoked `setHasSetInitialHTML` outside of `if` as done in recent changes